lib/mage/Mage.js

```diff
@@ -222,6 +222,10 @@ Mage.prototype.getTask = function () {
 Mage.prototype.setupCoreLibs = function () {
 	var core = this.core;
 
+	// Load the configuration
+
+	core.config = require('../config');
```

Comment:
Not sure what you mean by "this" - if you are asking whether config will be able to log, then the answer is yes.